Mathlib/Mathport/Syntax.lean

```diff
@@ -14,6 +14,7 @@ import Mathlib.Tactic.Ext
 import Mathlib.Tactic.Find
 import Mathlib.Tactic.LeftRight
 import Mathlib.Tactic.LibrarySearch
+import Mathlib.Tactic.MfldSetTac.Tactic
```

Comment:
Stuttering file namespace? You say "tactic" three times here. I think this tactic should go in or near `Logic.Equiv.LocalEquiv` since this is where it was found in lean 3.